notebooks/data_generator.py

```
@@ -0,0 +1,458 @@
+                    'oldbalanceOrg': 'Old Balance',
+                    'newbalanceOrig': 'New Balance',
+                    'nameDest': 'ID Dest',
+                    'isFraud': 'Is Fraud'
+                }, inplace=True)
+            except:
+                print("Path not found!")
+        elif self.data_name == 'fraud':
+            try: 
+                self.df = None
+
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)